chicago/permit_cleaning.py

```diff
@@ -301,28 +374,66 @@ def save_xlsx_files(df, max_rows, file_base_name):
     df_review_empty_invalid.to_excel(file_name_review_empty_invalid, index=False, engine="xlsxwriter")
 
 
+if __name__ == "__main__":
+    conn = connect(
+        s3_staging_dir=os.getenv(
+            "AWS_ATHENA_S3_STAGING_DIR",
+            "s3://ccao-athena-results-us-east-1",
+        ),
+        region_name=os.getenv(
+            "AWS_REGION",
+            "us-east-1",
+        ),
+    )
+    cursor = conn.cursor()
 
+    if os.path.exists("chicago_pin_universe.csv"):
+        print("Loading Chicago PIN universe data from csv.")
+        chicago_pin_universe = pd.read_csv(
+            "chicago_pin_universe.csv",
+            dtype={"pin": "string", "pin10": "string"}
+        )
```

Comment:
Previously we weren't specifying dtypes for the CSV, so pandas was inferring a float type for all of the PINs.